modules/mobile/spec/support/mobile_sm_client_helper.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require 'mobile/v0/messaging/client'
+
+module Mobile
+  module MessagingClientHelper
+    TOKEN = 'GkuX2OZ4dCE=48xrH6ObGXZ45ZAg70LBahi7CjswZe8SZGKMUVFIU88='
```

Comment:
Should this be a setting? Also confused about `used_id` below 😄, is that a placeholder?

Reply:
This is basically stubbing out the client step where it gets a session token; the value of token and client_id don't  matter because VCR  is just going  to match on the route. I'll make it a more obviously fake value.